scripts/generate_driver_wrappers.py

```diff
@@ -163,11 +163,11 @@ def main() -> int:
     """
     Main with command line arguments.
     """
-    def_arg_mbedtls_root = build_tree.guess_mbedtls_root()
+    def_arg_repo_root = build_tree.guess_mbedtls_root()
 
     parser = argparse.ArgumentParser()
-    parser.add_argument('--mbedtls-root', default=def_arg_mbedtls_root,
-                        help='root directory of mbedtls source code')
+    parser.add_argument('--repo-root', default=def_arg_repo_root,
```

Comment:
I don't think it's right to call it a “repo” root. That only makes sense when the source tree is a Git clone and not, for example, the result of unzipping an archive. “Project root”?

I'm ok with using “repo” internally in the script's code, but not in the interface (command line option and their documentation). But I think it would be clearer for maintenance to use the same terminology in the source code.

Reply:
I am happy to change to `project_root` throughout the script if that is preferred.

Reply:
let's go with `project_root`